src/customs/access.clj

```diff
@@ -76,11 +78,45 @@
    :body    body
    :headers {"Content-Type" "text/html; charset=utf-8"}})
 
```

Comment:
This is an embarassingly nit-picky comment, but here goes:

Sometime in between when @jalehman initially wrote this code and when our team expanded to include multiple engineers, we adopted [Readable Clojure](http://tonsky.me/blog/readable-clojure/) as our team's unofficial style guide.

One of the principles in Readable Clojure calls for [two lines of space between top-level forms](http://tonsky.me/blog/readable-clojure/#use-two-empty-lines-between-top-level-forms). The only exception we make to this rule is for `spec` - we only use one line of whitespace between a top-level and its corresponding spec definition.

It looks like the original commit didn't follow this rule, so it's totally reasonable that you would emulate the style found elsewhere in the file!

Would you mind bringing this file (and any others you see violating the "two line" rule) into compliance?

Reply:
Will do!